subjects/devops/array-selector/README.md

```diff
@@ -0,0 +1,95 @@
+## array-selector
+
+### Instructions
+
+Create a script `array-selector.sh`, which will have an array declared with the
+following values (in this order): `red`, `blue`, `green`, `white`, `black`.
+
+When executed, the script will try to print the element in the position specified in the first argument passed to the script.
+
+The script should return `Error` when:
```

Comment:
Just an opinion:
```
The script should return `Error` when:

- the number of given arguments is different from one,

- the given argument is not a number,

- the given argument is a number outside the range of the array.
```

Reply:
I like it better, I will change it like you suggested :)